pytype/tools/analyze_project/parse_args.py

```diff
@@ -9,6 +10,20 @@
 
 _ARG_PREFIX = '--'
 
+def _auto_detect_cpus():
```

Comment:
Could you add a second newline before this function, and before the class on line 28? (We put two blank lines between top-level functions and classes.)